dockerfiles/Dockerfile.openvino

```diff
@@ -36,11 +36,12 @@ RUN tar -xzf l_openvino_toolkit*.tgz && \
     cd - && \
     rm -rf l_openvino_toolkit* && \
     cd /opt/intel/openvino/install_dependencies && ./_install_all_dependencies.sh && dpkg -i *.deb && \
-    pip install networkx==2.3 test-generator==0.1.1 defusedxml>=0.5.0
+    pip install networkx==2.3 numpy>=1.12.0 test-generator==0.1.1 defusedxml>=0.5.0
```

Comment:
can you revert this line as well? numpy>=1.12.0 isn't required. numpy is already installed as dependency earlier.